Xamarin.Forms.Core/BindableNativeView.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms
+{
+	public abstract class BindableNativeView : View, INativeViewBindableController
```

Comment:
It's used in more places to call internal methods like when the target updates the value and we want to call set value here